### DISCUSSION TITLE
PSBTS for ordinals & fees

### DISCUSSION THREAD

**Discussion by 0xcielo** [paulmillr/micro-btc-signer#48] [Q&A]
How can I create a PSBTS for inscribe ordinals (svg format) and send a fee to another address for that? (btc) [ref](https://docs.xverse.app/sats-connect/creating-psbts)

**Comment by paulmillr** [comment]
Not sure. I did not do a deep-dive into ordinals yet.

**Comment by 0xcielo** [comment]
I am following this but i don't know how to create the psbt https://docs.xverse.app/sats-connect/creating-psbts :(

**Reply by foursmalltrees** [reply]
The thing is that xverse is using this library.

**Comment by RunlaiDeng** [comment]
> I am following this but i don't know how to create the psbt https://docs.xverse.app/sats-connect/creating-psbts :(

I try ,it will not send inscribtion to bitcoin node

**Comment by paulmillr** [comment]
Ordinals got easier with 1.3.0 https://github.com/paulmillr/scure-btc-signer/releases/tag/1.3.0
